### PR TITLE
Header and Router added to react code.

### PR DESCRIPTION
**Pull request by sadishag**
- Added react-router-dom to give pages 
- Added a start of the AddStock code as a sample "page"

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+  const displayMobile = () => {
+    const handleDrawerOpen = () =>
+      setViewState((prevState) => ({ ...prevState, drawerOpen: true }));
+    const handleDrawerClose = () =>
+      setViewState((prevState) => ({ ...prevState, drawerOpen: false }));
+    return (
+      <Toolbar>
+        <IconButton
+          {...{
```
Comment (k-dang) [inline src/components/Header.js]: do we need the spread on these props

Comment (sadishag) [reply src/components/Header.js]: Oops went full copy pasta there. I'll fix it tomorrow.

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
```
Comment (k-dang) [inline src/index.js:1]: wanna install prettier extension for vs code so we don't get formatting issues every time (ex the single vs double quotes)

Comment (sadishag) [reply src/index.js:1]: I'll add a .prettierrc file so that it doesn't pull random global settings.

**Review by k-dang — APPROVED** [review]

**Pull request merged**